onnxruntime/test/testdata/transformers/tiny_gpt2_beamsearch_node_stats.txt

```diff
@@ -0,0 +1,56 @@
+GptAttention_1_add,18432,0,0,0
+GptAttention_0_matmul,4096,0,0,0
+GptAttention_2_matmul,22528,0,0,0
+FullyConnect_MatMul_5,90112,0,0,0
+GptAttention_3,30720,0,36864,165888
+LayerNorm_4,18432,0,0,0
+GptAttention_1_matmul,22528,0,0,0
+FullyConnect_Add_5,18432,0,0,0
+GptAttention_2_add,18432,0,0,0
+FullyConnect_Add_3,18432,0,0,0
+GptAttention_3_add,18432,0,0,0
+Add_689,18432,0,0,0
+Add_886,18432,0,0,0
+LayerNorm_7,18432,0,0,0
+FullyConnect_MatMul_6,34816,0,0,0
+GptAttention_4,30720,0,36864,165888
+GptAttention_4_add,18432,0,0,0
+SkipLayerNormalization,18432,0,0,0
+LayerNorm_1,18432,0,0,0
+GptAttention_3_matmul,22528,0,0,0
+LayerNorm_8,18432,0,0,0
+FullyConnect_MatMul_8,34816,0,0,0
+FullyConnect_Add_7,18432,0,0,0
+LayerNorm_9,18432,0,0,0
+FastGelu_AddBias_3,73728,0,0,0
+FullyConnect_Add_1,18432,0,0,0
+GptAttention_4_matmul,22528,0,0,0
+GptAttention_0,13248,0,55296,165888
+FullyConnect_MatMul_2,34816,0,0,0
+FullyConnect_MatMul_9,90112,0,0,0
+MatMul_1165,146432,0,576000,0
+GptAttention_2,30720,0,36864,165888
+LayerNorm_6,18432,0,0,0
+BeamSearch_gpt2,24,0,256,1823244
```

Comment:
For a node with subgraph, does the resource include or exclude subgraphs?

Reply:
The resource does not include the subgraph nodes.
I will see to the subgraph names.